roles/etcd/tasks/main.yml

```diff
@@ -120,7 +120,9 @@
     enabled: yes
   register: start_result
 
-- include: etcdctl.yml
+- include_role:
+    name: etcd_common
+    tasks_from: etcdctl.yml
```

Comment:
Given
```yaml
- name: Install etcd
  package: name=etcd{{ '-' + etcd_version if etcd_version is defined else '' }} state=present
```

is in `etcd_server_certificates` as well, the etcd gets installed even before the `etcd` role itself is started. I will update the PR.